File: alpro-lioneljevon-9-10-2025/Versi C++/perkalian.cpp
#include <iostream>
#include <cstdio>
using namespace std;

int main() {
    cout << "TABEL PERKALIAN" << endl;

    int a, b;
    cout << "Masukkan angka pertama : "; cin >> a;
    cout << "Masukkan angka kedua   : "; cin >> b;

    cout << "\n📊 Hasil Tabel Perkalian dari " << a << " sampai " << b << endl;
    cout << "===================================================\n" << endl;

    int i = a;
    while(i <= b) {
        int j = a;
        while(j <= b) {
            printf("%2d x %-2d = %-4d  ", i, j, i*j);
            j++;
        }
        printf("\n");
        i++;
    }

    cout << "\n===================================================" << endl;

    return 0;
}

File: alpro-lioneljevon-9-10-2025/perkalian.py
print("TABEL PERKALIAN")


a = int(input("Masukkan angka pertama : "))
b = int(input("Masukkan angka kedua   : "))

print("\n📊 Hasil Tabel Perkalian dari", a, "sampai", b)
print("===================================================\n")

i = a
while i <= b:
    j = a
    while j <= b:
        print(f"{i:2} x {j:<2} = {i*j}", end="  ")
        j += 1
    print()
    i += 1

print("\n===================================================")


print("\n Versi Metode perulangan for")

a = int(input("Masukkan angka pertama : "))
b = int(input("Masukkan angka kedua   : "))

i = a
for i in range(b+1):
    j = a
    for j in range(b+1):
        print(f"{i:2} x {j:<2} = {i*j}", end="  ")
    print()

print("\n===================================================")
